src/main/java/org/opentripplanner/model/plan/StopArrival.java

```diff
@@ -20,11 +20,28 @@
      */
     public final Calendar departure;
 
+    /**
+     * For transit trips, the stop index (numbered from zero from the start of the trip).
+     */
+    public final Integer stopIndex;
+
+    /**
+     * For transit trips, the sequence number of the stop. Per GTFS, these numbers are increasing.
+     */
+    public final Integer stopSequence;
```

Comment:
Should this be named `gtfsStopSequence` ? It is just pass through information for gtfs?